### ISSUE TITLE
SPEAR data mismatch

### ISSUE THREAD

**Issue by wsalesky** [srophe/srophe-app-data#624]
There is a mismatch in 
http://syriaca.org/spear/3130

example:
```
            <div uri="http://syriaca.org/spear/3130-8">
                <listPerson>
                    <person>
                        <persName ref="http://syriaca.org/person/2408"/>
                        <sex value="M">Male</sex>
                    </person>
                </listPerson>
                <bibl type="primary">
                    <biblScope unit="pp">
                        <ptr target="#bibl3130-1"/>179</biblScope>
                </bibl>
            </div>
```

Back matter:
```
<bibl xml:id="bib3130-1">
                    <title xml:lang="en">The Sixth Book of the Select Letters of Severus, Patriarch of Antioch in the Syriac Version of Athanasius of Nisibis</title>
                    <title type="sub">Translation, I.1 - II.3</title>
                    <ptr target="http://syriaca.org/bibl/665"/>
                </bibl>
```

As you can see the above id uses 'bib' while the ptr target attributes all ref bibl.

**Comment by dlschwartz** [comment]
Sorry I let this one get by me Winona. I assume that bibl is correct as that matches our data model. Also, that is what the Chronicle has and there is less chance of error there. I had students doing some cutting and pasting and I'm guessing that's how the error crept in. Do you know if it is used throughout the letters? Would you be able to run a script correcting this throughout? Thanks Winona.

**Comment by wsalesky** [comment]
Okay, I updated these.

**Comment by dlschwartz** [comment]
With my new-found Oxygen skills, I guess I could have done this myself.
Thanks Winona!

On Fri, Oct 21, 2016 at 12:11 PM, Winona Salesky <notifications@github.com>
wrote:

> Closed #624
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_srophe_srophe-2Dapp-2Ddata_issues_624&d=CwMCaQ&c=ODFT-G5SujMiGrKuoJJjVg&r=QzRmQWHC9YV0c5AgrobproqvWFdcgVpw66BsP8UwAQ0&m=3OpJiPIlwCnMyVjYF-MedEu-VSVwEZ7M19FE4pJOjRQ&s=t3Ju2_YLc_Rw30BOsl5HOzZLhRk4mzoRXuP_uqPDLfU&e=>
> .
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_srophe_srophe-2Dapp-2Ddata_issues_624-23event-2D832298739&d=CwMCaQ&c=ODFT-G5SujMiGrKuoJJjVg&r=QzRmQWHC9YV0c5AgrobproqvWFdcgVpw66BsP8UwAQ0&m=3OpJiPIlwCnMyVjYF-MedEu-VSVwEZ7M19FE4pJOjRQ&s=58fGewwqllvYou6F8_QLCwfqwl2teJjS5ptrGclXd98&e=>,
> or mute the thread
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_AEwNzuUpTP8tfcx5ZOH87Tw3HPPNz-2DVXks5q2PIygaJpZM4KWEIX&d=CwMCaQ&c=ODFT-G5SujMiGrKuoJJjVg&r=QzRmQWHC9YV0c5AgrobproqvWFdcgVpw66BsP8UwAQ0&m=3OpJiPIlwCnMyVjYF-MedEu-VSVwEZ7M19FE4pJOjRQ&s=UZgDaRfjhZHCdR2vvfPrZI4HZeKAp2roA_vZPxY8l84&e=>
> .
>



-- 
Dr. Daniel L. Schwartz
Associate Professor of History
Associate Director of the Initiative for Digital Humanities, Media, and
Culture <http://idhmc.tamu.edu/>

Texas A&M University
4236 TAMU
College Station, Texas 77843-4236

*Paideia and Cult <http://chs.harvard.edu/CHS/article/display/5813>:
Christian Initiation in Theodore of Mopsuestia* (Center for Hellenic
Studies, 2013)
http;//syriaca.org <http://syriaca.org/>
